services/opencollective/opencollective-all.service.js

```diff
@@ -16,7 +16,7 @@ export default class OpencollectiveAll extends OpencollectiveBase {
     },
   }
 
-  static _cacheLength = 900
+  static _cacheLength = 1800
```

Comment:
Lets see if this gets us consistently under the limits. If not, I'll raise an issue with OC and see if they'd be willing to grant us a higher limit.